integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -63,17 +64,20 @@ def main(full_df):
     The main function. full_df is a report collection dataframe
```

Comment:
Looks like this isn't the entire "main" functionality of the script. Maybe move the bottom part of the script into this function to make it more main-like, or rename it something like `get_config_from_frequency_sweep`?